### ISSUE TITLE
Storage tests should use _serviceVersion for all client construction

### ISSUE THREAD

**Issue by tg-msft** [Azure/azure-sdk-for-net#9587]
Once the multi-versioned test support PR #9582 is merged, we should run through the tests and make sure every `*Client`/`*ClientOptions` ctor uses `_serviceVersion`.  98% of our tests use the default `GetOptions()` method in our base classes, but there a few random instances constructed elsewhere.

**Comment by JimSuplizio** [comment]
Hi @tg-msft, we deeply appreciate your input into this project. Regrettably, this issue has remained inactive for over 2 years, leading us to the decision to close it. We've implemented this policy to maintain the relevance of our issue queue and facilitate easier navigation for new contributors. If you still believe this topic requires attention, please feel free to create a new issue, referencing this one. Thank you for your understanding and ongoing support.
